app/src/main/java/com/xmartlabs/template/model/LoginRequest.kt

```diff
@@ -0,0 +1,4 @@
+package com.xmartlabs.template.model
+
+//TODO
+class LoginRequest
```

Comment:
It could, but that depends on the situation. Data classes don't have parameterless constructors, and that might clash with some library.